flood_adapt/object_model/hazard/hazard.py

```diff
@@ -13,10 +12,14 @@
     PhysicalProjection,
 )
 from flood_adapt.object_model.io.database_io import DatabaseIO
+from flood_adapt.object_model.projection import Projection
 
+# from flood_adapt.object_model.scenario import ScenarioModel
+from flood_adapt.object_model.strategy import Strategy
 
-class AttrModel(BaseModel):  # TODO replace with ScenarioModel
-    """BaseModel describing the expected variables and data types of the scenario"""
+
+class ScenarioModel(BaseModel):
```

Comment:
This should be in the scenarios.py script and then imported here. That would be the logical place

Reply:
see other comment